Assets/Scripts/Game/UI/DamageNumber.cs

```diff
@@ -0,0 +1,43 @@
+using UnityEngine.UI;
+using UnityEngine;
+using System.Collections;
+using TMPro;
+using Queue.Systems.UISystem;
+using DG.Tweening;
+
+
+public class DamageNumber : MonoBehaviour
+{
+    [SerializeField] TextMeshPro damageNumber;
+    [SerializeField] RectTransform rectTransform;
+    [SerializeField] RectTransform canvasRectTransform;
+    [SerializeField] float apearanceTime;
+    public bool isAvailable = true;
+
+    void Start()
+    {
+        isAvailable = true;
+    }
+
+    public void ActivateDamageNumber(float num, Vector3 pos)
+    {
+        Debug.Log($"Displaying damage number {num}");
+        isAvailable = false;
+        damageNumber.text = num.ToString();
+        pos = new Vector3(pos.x, pos.y + 5, pos.z);
```

Comment:
hardcoded value : You can add Vector3 offset as a serialized field instead of hardcoding "y+5"